images/win/Windows2016-Azure.json

```diff
@@ -498,12 +498,24 @@
                 "{{ template_dir }}/scripts/Installers/Install-JavaTools.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "scripts":[
+                "{{ template_dir }}/scripts/Tests/Validate-Java.ps1"
```

Comment:
This command won't run Pester. It will just run ps1 file as usual script. We should use syntax `Invoke-Pester Tests/Validate-MongoDB.ps1` to run pester tests on file.
Also I suggest to remove test invocations from template at all and run them from Installation files directly